spec/spec_helper.rb

```diff
@@ -14,7 +14,9 @@
 require "fakeredis/rspec"
 require "database_cleaner"
 require "simplecov"
-SimpleCov.start
+SimpleCov.start do
+  add_filter [ "/spec" ]
```

Comment:
I don't think we do this in any of our other projects.  Any reason we want to do this here?